src/mame/layout/smiconk.lay

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="overlay">
+		<rect>
+			<bounds x="0" y="0" width="1" height="240" />
+			<color red="1" green="1" blue="0.2" />
+		</rect>
+	</element>
+
+	<view name="Color Overlay">
+		<screen index="0">
+			<bounds left="0" top="0" right="3" bottom="4" />
+		</screen>
+		<element ref="overlay" blend="multiply">
+			<bounds left="0" top="0" right="3" bottom="4" />
+		</element>
```

Comment:
You should really define things in terms of `~scr0physicalxaspect~` and `~scr0physicalyaspect~`.

Reply:
I don't know, you'd just make it harder on yourself when you want to expand the artwork